src/components/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useContext } from 'react';
+
+import { TodosContext } from '../context';
+import { TodoFilter } from '../TodoFilter';
+import { TodoList } from '../TodoList';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+};
```

Comment:
looks like redundant

Reply:
In Misha's video the same, but I agree with you. Had no time to test(